messaging/src/main/java/org/axonframework/deadline/jobrunr/JobRunrDeadlineManager.java

```diff
@@ -0,0 +1,324 @@
+/*
+ * Copyright (c) 2010-2022. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.deadline.jobrunr;
+
+import org.axonframework.common.AxonConfigurationException;
+import org.axonframework.common.transaction.NoTransactionManager;
+import org.axonframework.common.transaction.TransactionManager;
+import org.axonframework.deadline.AbstractDeadlineManager;
+import org.axonframework.deadline.DeadlineException;
+import org.axonframework.deadline.DeadlineManager;
+import org.axonframework.deadline.DeadlineMessage;
+import org.axonframework.deadline.GenericDeadlineMessage;
+import org.axonframework.deadline.quartz.QuartzDeadlineManager;
+import org.axonframework.messaging.DefaultInterceptorChain;
+import org.axonframework.messaging.ExecutionException;
+import org.axonframework.messaging.InterceptorChain;
+import org.axonframework.messaging.ResultMessage;
+import org.axonframework.messaging.ScopeAwareProvider;
+import org.axonframework.messaging.ScopeDescriptor;
+import org.axonframework.messaging.unitofwork.DefaultUnitOfWork;
+import org.axonframework.messaging.unitofwork.UnitOfWork;
+import org.axonframework.serialization.Serializer;
+import org.axonframework.serialization.SimpleSerializedObject;
+import org.axonframework.tracing.NoOpSpanFactory;
+import org.axonframework.tracing.Span;
+import org.axonframework.tracing.SpanFactory;
+import org.jobrunr.scheduling.JobScheduler;
+import org.slf4j.Logger;
+
+import java.time.Instant;
+import java.util.UUID;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+import static java.lang.String.format;
+import static org.axonframework.common.BuilderUtils.assertNonNull;
+import static org.axonframework.deadline.GenericDeadlineMessage.asDeadlineMessage;
+import static org.slf4j.LoggerFactory.getLogger;
+
+/**
+ * Implementation of {@link DeadlineManager} that delegates scheduling and triggering to a Jobrunr
+ * {@link JobScheduler}.
+ *
+ * @author Tom de Backer
+ * @author Gerard Klijs
+ * @since 4.7.0
+ */
+public class JobRunrDeadlineManager extends AbstractDeadlineManager {
+
+    private static final Logger LOGGER = getLogger(JobRunrDeadlineManager.class);
```

Comment:
Super duper nit: the syntax for loggers throughout AF is lowercase:
```suggestion
    private static final Logger logger = getLogger(JobRunrDeadlineManager.class);
```

Reply:
Changed